packages/api/src/config/brevo-module.config.ts

```diff
@@ -1,5 +1,8 @@
+import { BrevoContactAttributesInterface } from "src/types";
+
 export interface BrevoModuleConfig {
     brevo: {
         apiKey: string;
+        BrevoContactAttributes: BrevoContactAttributesInterface;
```

Comment:
should the attributes be required?
in the packages/api/src/brevo-api/brevo-api-contact.service.ts and packages/api/src/brevo-contact/dto/brevo-contact.factory.ts the attributes are optional.

I think they could be optional, but if not I'd say they should be required in the 2 files mentioned above.